<launch>
  <arg name="input_topic_name" default="/sensing/lidar/top/pointcloud_raw"/>
  <arg name="output_topic_name" default="/sensing/lidar/top/pickup_based_voxel_grid_downsample_filter/pointcloud"/>
  <arg name="input_frame" default=""/>
  <arg name="output_frame" default=""/>
  <arg name="pickup_based_voxel_grid_downsample_filter_param_file" default="$(find-pkg-share autoware_pointcloud_preprocessor)/config/pickup_based_voxel_grid_downsample_filter_node.param.yaml"/>
  <node pkg="autoware_pointcloud_preprocessor" exec="pickup_based_voxel_grid_downsample_filter_node" name="pickup_based_voxel_grid_downsample_filter_node">
    <param from="$(var pickup_based_voxel_grid_downsample_filter_param_file)"/>
    <remap from="input" to="$(var input_topic_name)"/>
    <remap from="output" to="$(var output_topic_name)"/>
    <param name="input_frame" value="$(var input_frame)"/>
    <param name="output_frame" value="$(var output_frame)"/>
  </node>
</launch>
